Makefile

```diff
@@ -0,0 +1,55 @@
+DEPDIR=$(PWD)/deps
+BUILDDIR=$(PWD)/build
+
+LIBERASURE=$(DEPDIR)/liberasurecode
+ISALSRC=$(DEPDIR)/isa-l
+DEV_1_0 := $(shell git show origin/development/1.0 | grep commit | cut -d" " -f 2)
+
+.PHONY: default test clean pretty cmds
+
+default: $(BUILDDIR)/lib/liberasurecode.a $(BUILDDIR)/lib/libisal.a cmds
+	PKG_CONFIG_PATH=$(BUILDDIR)/lib/pkgconfig \
+	go build
+
+test: $(BUILDDIR)/lib/liberasurecode.a $(BUILDDIR)/lib/libisal.a
+	DYLIB_LIBRARY_PATH=$(BUILDDIR)/lib \
+	LD_LIBRARY_PATH=$(BUILDDIR)/lib \
+	PKG_CONFIG_PATH=$(BUILDDIR)/lib/pkgconfig \
+	go test -v .
+
+lint:
+	golangci-lint run --new-from-rev ${DEV_1_0}
+
+cmds: ec-split ec-info
+
+ec-split: $(PWD)/cmd/ec-split/main.go $(PWD)/backend.go $(PWD)/streaming.go
+	PKG_CONFIG_PATH=$(BUILDDIR)/lib/pkgconfig \
+	go build github.com/scality/erasurecode/cmd/ec-split
+
+ec-info: $(PWD)/cmd/ec-info/main.go $(PWD)/backend.go $(PWD)/streaming.go
+	PKG_CONFIG_PATH=$(BUILDDIR)/lib/pkgconfig \
+	go build github.com/scality/erasurecode/cmd/ec-info
+
+clean:
+	rm -rf $(BUILDDIR) $(DEPDIR)
+
+pretty:
+	find $(PWD) -name '*.go' | xargs gofmt -l -w
+
+$(ISALSRC)/autogen.sh:
+	git clone --depth 1 --branch v2.28.0 https://github.com/intel/isa-l.git $(ISALSRC)
+
+$(LIBERASURE)/autogen.sh:
+	git clone --depth 1 --branch v1.6.1 https://github.com/scality/liberasurecode.git $(LIBERASURE)
```

Comment:
Assuming 1.6.1 == afc38de0498bf8ca7f5ef994431a2e85549b008e, there

Reply:
it is: `commit afc38de0498bf8ca7f5ef994431a2e85549b008e (HEAD -> development/1.0, tag: v1.6.1`